utils/make-phar.php

```diff
@@ -0,0 +1,2 @@
+<?php
+	include( '../version.mustache' );
```

Comment:
This is not needed here and should be done in the `wp-cli/wp-cli` package instead.